BistroQ.Services/Mappings/MappingProfile.cs

```diff
@@ -17,13 +19,31 @@ public MappingProfile()
     {
         CreateMap<Product, ProductDto>().ReverseMap();
         CreateMap<CreateProductRequestDto, Product>()
```

Comment:
Why was `AfterMap` used here? `AutoMapper` already supports nested mapping. 
Could it be that `ConstructUsing` disables this functionality?

Reply:
To fix the error with request form-data, I have to map every nested items to top-level. Nested mapping does not worked.

Reply:
> To fix the error with request form-data, I have to map every nested items to top-level. Nested mapping does not worked.

```csharp
public class CreateProductRequestDto
{
    public string Name { get; set; }
    public decimal Price { get; set; }
    public string Unit { get; set; }
    public decimal DiscountPrice { get; set; }
    public int? CategoryId { get; set; }
    public double Calories { get; set; }
    public double Fat { get; set; }
    public double Fiber { get; set; }
    public double Protein { get; set; }
}
```

`NutritionFact` related fields does not belongs to any object, as you can see.

Reply:
Ok